content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -120,12 +120,11 @@ Once you have a Linux-based Kubernetes control-plane node you are ready to choos
 
     {{< note >}}
     If you're using a different interface rather than Ethernet (i.e. "Ethernet0 2") on the Windows nodes, you have to modify the line:
-    
+     {{< /note >}}
```

Comment:
this is not entirely accurate now either. can you please put this in the note section?
<<
If you’re using a different interface rather than Ethernet (i.e. “Ethernet0 2”) on the Windows nodes, you have to modify the line below in the flannel-host-gw.yml or flannel-overlay.yml file and specify your interface accordingly. 
>>

Reply:
Preview looks correct with formatting
https://deploy-preview-20132--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/